src/GitHub.InlineReviews/Tags/InlineCommentTagger.cs

```diff
@@ -187,16 +185,21 @@ void Initialize()
 
             if (session == null)
             {
-                managerSubscription = sessionManager.WhenAnyValue(x => x.CurrentSession).Subscribe(SessionChanged);
+                managerSubscription = sessionManager.WhenAnyValue(x => x.CurrentSession).Subscribe(x => ForgetWithLogging(SessionChanged(x)));
             }
             else
             {
-                SessionChanged(session);
+                ForgetWithLogging(SessionChanged(session));
             }
 
             initialized = true;
         }
 
+        static void ForgetWithLogging(Task task)
```

Comment:
I looked into this but the project with the `VsOutputLogger` didn't appear to be referenced.  That would however be my preference (I don't like ignoring exceptions). Suggestions welcome?